nodejs/src/access_token_common.js

```diff
@@ -7,11 +7,10 @@ import {ApiCommon} from './api_common.js'
 export class AccessTokenCommon extends ApiCommon {
 
   constructor(api_config, {name = null}) {
```

Comment:
@davidchaiken this could simply be {name} as it will be undefined if not found in the input object.

Reply:
Unless null is a valid assignment when not provided, then you should keep it as is.

Reply:
I removed all of the instances of argument initialization to null, except where null is a valid assignment (in the case of v3 scopes).

Reply:
Oops! This change didn't work out for me. I'm getting this error:
```TypeError: Cannot destructure property 'name' of 'undefined' as it is undefined.```

In the cases where I specified null, null is a valid assignment when not provided.